util/dvsim/FlowCfg.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
         self.branch = args.branch
         self.job_prefix = args.job_prefix
         self.gui = args.gui
+        self.batch_mode_prefix = "" if (args.gui) else "-batch"
```

Comment:
Nit: no need for the parens here

Reply:
Oops, I should put it in `FormalCfg`. And thank you for the suggestion :)